clang/lib/Sema/SemaCodeComplete.cpp

```diff
@@ -2186,6 +2197,45 @@ AddOrdinaryNameResults(SemaCodeCompletion::ParserCompletionContext CCC,
       } else {
         Results.AddResult(Result("template", CodeCompletionResult::RK_Keyword));
       }
+
+      if (SemaRef.getLangOpts().CPlusPlus20 &&
+          SemaRef.getLangOpts().CPlusPlusModules) {
+        // export
+        Results.AddResult(Result("export", CodeCompletionResult::RK_Keyword));
+
+        if (SemaRef.CurContext->isTranslationUnit()) {
```

Comment:
Should we restrict that using `getCurrentModule()->Kind`  @ChuanqiXu9 ?

Reply:
i think we can't do it here. Since we don't the kind of the current module before we see module declarations.

Reply:
Well, exactly.
If we haven't see `module;`, we should propose `module;` and `export module`
If we have seen `module;` - we should not propose it again
If we have seem `export module`, we should not propose `import`

Reply:
It only happens during completion so there is probably a bug with that https://godbolt.org/z/7M8EPodoP

Reply:
I made some modification to `ParseExportDeclaration`  to fix the problem.